### ISSUE TITLE
Warning for misnamed sub-objects

### ISSUE THREAD

**Issue by SerialMascot** [Baezon/pof-tools#142]
I saw a red warning  on a skybox's sub-model. The warning said that the sub-object needed a name, but I foolishly gave it the wrong name, and this caused crashed without helpful logs. Changing the name to "detail0" solved the problem. Maybe another red warning can be added when a sub-model has the _wrong_ name. I'm not sure if the skybox is a special case for sub-object naming...

**Comment by Baezon** [comment]
Indeed, crashing for any reason is an FSO bug and should be also reported there, if you're certain it was the name? What name did you give it?
